tests/resources/spec.json

```diff
@@ -593,29 +603,86 @@
         ]
       }
     },
+    "/mock/tests/general/400-error": {
+      "get": {
+        "summary": "Mock a an 400 failed request",
+        "operationId": "generalError400",
+        "consumes": ["application/json"],
+        "produces": [],
+        "tags": ["general"],
+        "description": "",
+        "responses": {
+          "500": { "description": "File", "schema": { "type": "file" } }
```

Comment:
Response, Description and schema seems wrong, this should be 400 not 500 and not file

Reply:
👍🏻 I'll fix that